test/index.test.ts

```diff
@@ -19,8 +20,32 @@ describe("Testing Toucan-SDK", function () {
   let addr2: SignerWithAddress;
   let toucan: ToucanClient;
   let swapper: Contract;
-  let scoredTCO2sBCT: string[];
-  let scoredTCO2sNCT: string[];
+
+  /**
+   *
+   * @param poolSymbol Pool symbol to get scored TCO2s for
+   * @returns an array of addresses representing TCO2s in the given pool that have a balance > 1.0
+   */
+  const getFilteredScoredTCO2s = async (
+    poolSymbol: PoolSymbol
+  ): Promise<string[]> => {
+    const scoredTCO2s = await toucan.getScoredTCO2s(poolSymbol);
+    const pool = toucan.getPoolContract(poolSymbol);
+    return await asyncFilter(scoredTCO2s, async (tco2: string) => {
+      // tokenBalances is not defined because the storage contract is not in typechain
+      // @ts-ignore
```

Comment:
couldn't you do `pool['tokenBalances']` to circumvent this `ts-ignore`? or otherwhise get the storage contract into typechain?